modules/profiles/functions.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+if (!defined('DEBUG_MODE')) { die(); }
+
+if (!hm_exists('add_profile')) {
+    function add_profile($name, $signature, $reply_to, $is_default, $email, $server_mail, $smtp_server_id, $imap_server_id, $context) {
+        $profile = array(
+            'name' => $name,
+            'sig' => $signature,
+            'smtp_id' => $smtp_server_id,
+            'replyto' => $reply_to,
+            'default' => $is_default,
+            'address' => $email,
+            'server' =>  $server_mail,
+            'user' => $email,
+            'type' => 'imap'
+        );
+
+        Hm_Profiles::init($context);
```

Comment:
Please use the Hm_Handler_profile_data module to do this. Just include that module in the module queue on the page(s) you want to use profiles.